ui-charts/src/speedSpaceChart/index.ts

```diff
@@ -6,3 +6,4 @@ export {
   type SpeedSpaceChartProps,
 } from './components/SpeedSpaceChart';
 export type { Data as SpeedSpaceChartData } from './types';
+export * from './types';
```

Comment:
I think this is a bit too wide of an export: this will export as public API a whole bunch of our internal types, such as `Store`.

Reply:
ok, I restricted to only those used by osrd front